src/NuGet.Core/NuGet.PackageManagement/NuGet.PackageManagement.csproj

```
@@ -28,6 +28,10 @@
     <Reference Include="Microsoft.CSharp" />
   </ItemGroup>
   
+  <ItemGroup>
+    <PackageReference Include="System.Threading.Tasks.Dataflow" />
```

.NET Core has TPL dataflow "built-in". THere's no need for a package, and in particular anyone using `NuGet.PackageManagement` in their own .NET Core project doesn't need to download an additional package as a dependency.

```suggestion
    <PackageReference Include="System.Threading.Tasks.Dataflow" Condition=" '$(TargetFrameworkIdentifier)' = '.NETFramework' " />
```

---

Ok. Taken the suggestion.